Filters/exceptions.txt

```diff
@@ -1,3 +1,5 @@
+! https://github.com/AdguardTeam/AdGuardSDNSFilter/issues/1599
+@@||cameraprive.com^|
```

Comment:
Hm, should it be `||cameraprive.com^` in another file? @Alex-302 
https://github.com/AdguardTeam/HostlistsRegistry/blob/main/filters/general/filter_1_DnsFilter/filter.txt#L35470

Reply:
Yes.

@piquark6046  Run build to check if it works correctly.